### ISSUE TITLE
FFT support 

### ISSUE THREAD

**Issue by jakirkham** [Theano/libgpuarray#463]
Just opening this to note interest in having FFT support. Is this something that is planned at some point?

**Comment by nouiz** [comment]
In Theano, we wrapped cufft. We won't manually implement an fft here. You
can use skcuda cufft wrapper with libgpuarray as we done in Theano.
Do you want to use cuda or OpenCL? Do you know and OpenCL try
implementation?

**Comment by jakirkham** [comment]
I see. Wasn't sure exactly what all was being considered in scope for this project. Generally got the impression that GPU array computations from Theano were being offloaded to libgpuarray/pygpu.

Ideally was wanting to use both CUDA and OpenCL. As far as OpenCL implementations, [clFFT]( https://github.com/clMathLibraries/clFFT ) comes to mind.

**Comment by jakirkham** [comment]
To be clear, wasn't expecting FFTs to be implemented in this library. More than happy to have an interface to cuFFT and clFFT as was done with BLAS.

**Comment by nouiz** [comment]
I agree, an interface that wrap cufft and clfft fit in this project.
Contribution welcome. It is currently not on our short term project we plan
to work on.
